docs/http_api.rst

```diff
@@ -309,10 +315,8 @@ Status events ``POST``\ed to the ``status_url`` specified in :http:post:`/channe
       status: "major",
       component: "smpp",
       channel_id: "channel-uuid-5678",
-      reasons: [{
-        type: "connection_lost",
-        message: "Connection lost"
-      }],
+      type: "connection_lost",
+      message: "Connection lost",
```

Comment:
This change does mean that `type` and `message` are now mandatory fields for things publishing status events, but I think that is maybe a reasonable thing to enforce.